src/bin/add/args.rs

```diff
@@ -96,10 +95,11 @@ impl Args {
                 } else if let Some(ref path) = self.flag_path {
                     dependency.set_path(path)
                 } else {
+                    let dep = try!(get_latest_dependency(&self.arg_crate));
                     let v = format!("{prefix}{version}",
                                     prefix = self.get_upgrade_prefix().unwrap_or(""),
-                                    version = try!(get_latest_version(&self.arg_crate)));
-                    dependency.set_version(&v)
+                                    version = dep.version().unwrap_or_else(|| unreachable!()));
```

Comment:
Can you add a comment why you are sure this is actually always present and unreachable is fine here?